openpype/hosts/fusion/deploy/fusion_shared.prefs

```diff
@@ -1,19 +1,20 @@
 {
 Locked = true,
 Global = {
- Paths = {
-  Map = {
-   ["OpenPype:"] = "$(OPENPYPE_FUSION)/deploy",
-   ["Reactor:"] = "$(REACTOR)",
-
-   ["Config:"] = "UserPaths:Config;OpenPype:Config",
-   ["Scripts:"] = "UserPaths:Scripts;Reactor:System/Scripts;OpenPype:Scripts",
-   ["UserPaths:"] = "UserData:;AllData:;Fusion:;Reactor:Deploy"
+  Paths = {
+    Map = {
+      ["OpenPype:"] = "$(OPENPYPE_FUSION)/deploy",
+      ["Config:"] = "UserPaths:Config;OpenPype:Config",
+      ["Scripts:"] = "UserPaths:Scripts;Reactor:System/Scripts;OpenPype:Scripts",
+    },
   },
- },
- Script = {
-			PythonVersion = 3,
-			Python3Forced = true
-		},
+  Script = {
+    PythonVersion = 3,
+    Python3Forced = true
+    },
+  UserInterface = {
```

Comment:
I know nothing about fusion but why is `UserInterface` filled? It looks like forcing a language and style.

Reply:
Yeah, I noted that in description. If the profile is generated by fusion (i.e. the copy profile isn't toggled), all Fusion setting will be set to default, including the language interface. Menu localization is recent BMD `innovation`. I know very few people who uses native language UI interface. Besides, if the native locale is set up (in my case it is Russian), the custom Fusion profile directory variable is not read. This is probably a fusion bug. So this is seem to be an essential fix.

Reply:
Since we're now copying the prefs anyway, should the master prefs maybe be 'generated' along with the copied prefs and maybe get generated e.g. per project from e.g. OpenPype Settings instead. Looking at what is being altered in this PR I have a feeling it might be worth being able to just customize the master prefs directly from OpenPype settings?
We would basically deploy from settings into the copied preferences folder.

Thoughts?

Reply:
We do that with `workspace.mel` in Maya, it could have benefits. But in that case we should handle adding the core integration line:
```
["OpenPype:"] = "$(OPENPYPE_FUSION)/deploy",
```
For cases we would change name or path?